src/dom/dom.js

```diff
@@ -953,7 +953,7 @@ p5.prototype.createRadio = function() {
       }
     } else {
       for (const option of self._getOptionsArray()) {
-        if (option.value === value) {
+        if (option.value === value.toString()) {
```

Comment:
I'm having second thought about this as it doesn't work in all scenarios. I think it may be better to note in the documentation recommending only string values be used for `.option` and `.selected`. But that's just what I'm thinking right now, not sure if it's the best solution here.

Reply:
My thought process about this was: since `option.value` will always be a string, we could check it strictly with a string too. This way, the user can put a string, integer or character value as the `value` and it will convert it to a string and then make comparisons with the original value, which was automatically converted to a string by the HTML.
Could you help me with figuring out where this might not work? 
Thanks for replying!

Reply:
`null` does not work and all objects will get casted into `[object Object]`. If the documentation does not make clear the expected value, casted or otherwise, would be, implicit casting into strings may not be enough to avoid potential confusion.

Reply:
I understand the problem now. I will make changes to the documentation to resolve this issue, and make a pull request.
Appreciate the help!